cmd/api-linter/cli.go

```diff
@@ -63,19 +59,25 @@ func runCLI(rules lint.Rules, configs lint.Configs, args []string) error {
 			},
 			Action: func(c *cli.Context) error {
 				filenames := c.Args()
+
+				// Sanity check: Were we given any files to parse at all?
+				// If not, abort.
```

Comment:
Add a TODO to inform the user to include files, or better that we check all proto files in the current directory.

Reply:
Added an error message.